cpp/modmesh/serialization/SerializableItem.hpp

```diff
@@ -170,6 +170,29 @@ std::string JsonHelper::to_json_string(const T & value)
         oss << "]";
         return oss.str();
     }
+    else if constexpr (is_specialization_of_v<std::unordered_map, T>)
+    {
+        static_assert(std::is_same_v<typename T::key_type, std::string>, "Only support std::unordered_map<std::string, ...>.");
+
+        std::vector<std::string> keys;
+        for (const auto & kv : value)
+        {
+            keys.push_back(kv.first);
+        }
+        std::sort(keys.begin(), keys.end());
```

Comment:
sort the key to ensure output consistency in all platforms.

Reply:
It's not unusual to use a specific order (like that of key insertion).  Could you please add a code comment indicating the possibility (for future enhancement)?

Reply:
Indeed, sorting seems to be unnecessary.  This is more for testing. For now, it shouldn't be a bottleneck, so let me just add a comment here.